package/MDAnalysis/coordinates/CRD.py

```diff
@@ -151,11 +154,21 @@ def __init__(self, filename, **kwargs):
         ----------
         filename : str or :class:`~MDAnalysis.lib.util.NamedStream`
              name of the output file or a stream
+
+        extended : bool (optional)
+             By default, noextended CRD format is used [``False``].
+             However, extended CRD format can be forced by
+             specifying `extended` ``=True``.
```

Comment:
Make clear that we still use EXT of necessary.
```suggestion
             specifying `extended` ``=True``. Note that the extended format
             is *always* used if the number of atoms exceeds 99,999, regardless
             of the setting of `extended`.
```